docs/install/install-server/configure-server.inc

```diff
@@ -74,8 +79,21 @@ ssh and if the remote agent is used.
 4. Test if you can login into a machine that has the public key and make sure ssh does not show you any prompts to store
    the host key.
 
+Step 7: Connect to the dashboard
```

Comment:
I don't like the title. When I read it, I would expect that the user will be connecting to the dashboard in the step of the installation procedure, but that's not true.